scripts/control_device.py

```diff
@@ -75,6 +75,13 @@
     required=True,
     help="the identification of the device",
 )
+shared_parser.add_argument(
+    "-k",
+    "--device-key",
+    type=str,
+    required=True,
```

Comment:
Can we make this optional? I have checked and my devices doesn't require a key, we can add the logic in the API to use the key if user supplied it, if not just use "00".
The script can make this parameter optional with a default value, with this you can also just give a single example of usage with a key instead of modifying all examples

Reply:
Sure. Yes, you are right we can make it optional for this script.
I will make the changes.

Reply:
Changes requested done.